[feature] Add lines opening and closing times (#626)
CanalTP/transit_model: src/documentation/common.md

       `[name of origin's stop area] - [name of destination's stop area]`
  - the `route.destination_id` is set (overridden if needed) with the
    destination's stop area selected with the above rule

> - If a line has no opening and closing times, then they are generated.
>   - if one of the two values (at least) is empty, then both are recalculated, otherwise they are kept.

Reviewer: ```suggestion
- If a line has an empty opening or closing times, then they are both generated.
```

Author: Commited :+1:
